boards/arm/cy8ckit_062_pioneer/Kconfig.board

```
@@ -0,0 +1,6 @@
+# SPDX-License-Identifier: Apache-2.0
+# Copyright (c) 2023 David Ullmann
+
+config BOARD_CY8CKIT_062_PIONEER_M4
+	bool "PSoC6 BLE Pioneer Kit"
```

There are multiple BLE kits. I suggest:
```suggestion
	bool "PSoC6 S4 BLE Pioneer Kit"
```

---

ahh sorry I put the wrong name here completely. It's not a ble kit at all. I'm changing to what's on the box exactly.